boards/arm/qemu_cortex_a53/board.cmake

```diff
@@ -5,9 +5,16 @@ set(EMU_PLATFORM qemu)
 set(QEMU_ARCH aarch64)
 
 set(QEMU_CPU_TYPE_${ARCH} cortex-a53)
+
+if(CONFIG_ARMV8_A_NS)
+set(QEMU_MACH virt,virtualization=on,gic-version=3)
```

Comment:
This is confusing because `CONFIG_ARMV8_A_NS` has nothing to do with you change here. You removed `secure=on` and added `virtualization=on`, this means that QEMU is now starting from EL2. 

- What's the relation with `NS/S` world?

My expectation here is that you can have QEMU running Zephyr in `NS` world but this is not the case because when you enable `CONFIG_ARMV8_A_NS` all the QEMU tests stop working (because starting from EL2 is not supported).

- What is this change for considering that `CONFIG_ARMV8_A_NS` is actually breaking the QEMU emulation?